code/modules/synthesized_instruments/real_instruments.dm

```diff
@@ -283,7 +283,8 @@
 
 
 /obj/item/synthesized_instrument/ui_interact(mob/user, ui_key = "instrument", var/datum/nanoui/ui = null, var/force_open = 0)
-	real_instrument.ui_call(user,ui_key,ui,force_open)
+	if (real_instrument)
+		real_instrument.ui_call(user,ui_key,ui,force_open)
```

Comment:
This proc should not be called on deleted instruments: if it is, that's an explicit call something is making illegally. Are there other ways `real_instrument` can be null?